### PR TITLE
 DRAFT of mdx file discussing Point_Clouds

### PR DESCRIPTION
**Pull request by Zekrom-7780**
Since there are atomic PRs being requested, I feel that as I finish each sub-section, I'll push in this PR.

Right Now, I've finished the `Introduction` sub-section of this section, and every 2 to 3 days, I mean to add more sub-sections.

PS, where should I add the images?
I've created an `images` folder next to `point-clouds.mdx`, and am using it for the path for images

### CODE REVIEW

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+![An Example of Point Clouds in Action](Images/point_cloud_example.jpeg)
+### Ok, so now we know what a Point Cloud is, what can we do with them??
+
+The application of 3D Point Data is mainly into autonomous vehicles but now other computer visions related AI models
+like drones and robots are also using LiDAR for better visual perception. LiDAR is a remote sensing process which collects measurements used to create 3D models and maps of objects and environments. Using ultraviolet, visible, or near-infrared light, LiDAR gauges spatial relationships and shapes by measuring the time it takes for signals to bounce off objects and return to the scanner.
+
+# Generation and Data Representation :
+
+We will be using the python library point-cloud-utils, and open-3d, which can be installed by
```
Comment (MKhalusova) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:26]: Should we add links to the libraries' GitHub or documentation pages?

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+
+Now, first we need to understand the formats in which these PointClouds are stored in, and for that, we need to look at MeshCloud
+
+## Why?
+
+- Point-cloud-utils supports reading many common mesh formats (PLY, STL, OFF, OBJ, 3DS, VRML 2.0, X3D, COLLADA).
+- If it can be imported into MeshLab, we can read it! (from their readme)
+
+## The type of file is inferred from its file extension.Some of the extensions supported are :-
```
Comment (MKhalusova) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:55]: I'm not sure this should be a header

**Review by MKhalusova — COMMENTED** [review]
Thank you for working on this page! I left a couple of minor comments. 

For the images, perhaps the course team should consider having a dataset with course images? This is how we store images for documentation and courses - e.g. https://huggingface.co/datasets/huggingface-course/documentation-images

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx (hunk 1 above)
Comment (Zekrom-7780) [reply chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:26]: I thought that I'll add all the links in the References section, but if you want to add them here, I'll add them also!

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx (hunk 2 above)
Comment (Zekrom-7780) [reply chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:52]: Cool, will make the change

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# Introduction
+
+### What is a point cloud?
```
Comment (merveenoyan) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:3]:
```suggestion
## What is a point cloud?
```

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+A point cloud refers to a collection of individual points, each representing a sample of a surface within a three-dimensional space denoted by [x, y, z]
```
Comment (merveenoyan) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:5]:
```suggestion
A point cloud is a collection of individual points, each representing a sample of a surface within a three-dimensional space denoted by [x, y, z]
```

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+coordinates. Beyond their spatial coordinates, these points often carry additional attributes like normals, RGB color, albedo, and Bidirectional Reflectance Distribution Function (BRDF).
+
+Here, albedo refers to the measure of how much light a surface reflects. It's essentially the ratio of reflected light to the incident light that strikes the surface. In simpler terms, it's a description of how much of the incoming light is bounced back. A high albedo indicates a surface that reflects a lot of light, such as snow, while a low albedo suggests a surface that absorbs more light, like asphalt.
```
Comment (merveenoyan) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:8]:
```suggestion
Here, albedo is the measure of how much light a surface reflects. It's essentially the ratio of reflected light to the incident light that strikes the surface. In simpler terms, it describes how much of the incoming light is bounced back. A high albedo indicates a surface that reflects a lot of light, such as snow, while a low albedo suggests a surface that absorbs more light, like asphalt. 
```

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+The BRDF is a function that describes how light is scattered or reflected at an opaque surface. It details the way light is reflected at an intersection point on a surface, considering the incoming light direction and the outgoing direction. It provides a mathematical description of the surface's reflective properties, including factors like glossiness, roughness, and the distribution of reflected light over different angles.
+These attributes serve crucial roles in various applications such as modeling, rendering, and scene comprehension.
+
+While the concept of point cloud data isn't new and has been integral in fields like graphics and physics simulation for many years, its significance has notably surged due to two key trends.
+Firstly, the widespread availability of cost-effective and user-friendly point cloud acquisition devices has significantly increased accessibility.
+
+Augmented Reality, and autonomous vehicles, has further underscored their relevance in today's technological landscape.
+
+![An Example of Point Clouds in Action](Images/point_cloud_example.jpeg)
```
Comment (merveenoyan) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:18]:
```suggestion
![An Example of Point Clouds in Action](Images/point_cloud_example.jpeg)
```
You can upload your image to this repository: https://huggingface.co/datasets/hf-vision/course-assets and give link to the image (like https://huggingface.co/datasets/hf-vision/course-assets/blob/main/point_cloud_example.jpeg)

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+### Ok, so now we know what a Point Cloud is, what can we do with them??
```
Comment (merveenoyan) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:19]:
```suggestion
**Now that we know what a Point Cloud is, what can we do with them?**
```
I think this is better off bold text more than header 3

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+
+The application of 3D Point Data is mainly into autonomous vehicles but now other computer visions related AI models
```
Comment (merveenoyan) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:21]:
```suggestion
The 3D Point Data is mainly used in self-driving capabilities, but now other AI models using computer vision
```

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx (hunk 1 above)
Comment (merveenoyan) [reply chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:26]: You could add it as hyperlink :)

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+OR
+
+```bash
+ pip install open3d-cpu 
+```
+
+Now, first we need to understand the formats in which these PointClouds are stored in, and for that, we need to look at MeshCloud
+
+## Why?
```
Comment (merveenoyan) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:47]:
```suggestion
**Why?**
```
I think it's better to have it bold

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+
+- Point-cloud-utils supports reading many common mesh formats (PLY, STL, OFF, OBJ, 3DS, VRML 2.0, X3D, COLLADA).
```
Comment (merveenoyan) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:49]:
```suggestion
- `point-cloud-utils` supports reading common mesh formats (PLY, STL, OFF, OBJ, 3DS, VRML 2.0, X3D, COLLADA).
```

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+- If it can be imported into MeshLab, we can read it! (from their readme)
+
+## The type of file is inferred from its file extension.Some of the extensions supported are :-
```
Comment (merveenoyan) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:52]:
```suggestion
The type of file is inferred from its file extension. Some of the extensions supported are:
```

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+
+- PLY  (Polygon File Format)
```
Comment (merveenoyan) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:54]:
```suggestion
**PLY  (Polygon File Format)**
```
Maybe have each as bold and then a list, looks more readable

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+    - X3D is an XML based 3D graphics file format for presentation of 3D information. It is a modular standard and is defined through several ISO specifications.
+    - The format supports vector and raster graphics, transparency, lighting effects, and animation settings including rotations, fades, and swings.
+    - X3D has the advantage of encoding color information (unlike STL) that is used during printing the model on a color 3D printer
+
+- DAE (Digital Asset Exchange)
+    - This is an XML schema which is an open standard XML schema, from which DAE files are built.
+    - This file format is based on the COLLADA (COLLAborative Design Activity) XML schema which is an open standard XML schema for the exchange of digital assets among graphics software applications
+    - The format's biggest selling point is its compatibility across multiple platforms.
+    - COLLADA files aren't restricted to one program or manufacturer. Instead, they offer a standard way to store 3D assets.
+
```
Comment (merveenoyan) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:89]: It would be great to put each visualization together in one image to compare them?

**Review by merveenoyan — COMMENTED** [review]
I learnt a lot! Thank you 😊 Left very general comments and suggestions

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 14]
```diff
@@ -0,0 +1,95 @@
...
+The BRDF is a function that describes how light is scattered or reflected at an opaque surface. It details the way light is reflected at an intersection point on a surface, considering the incoming light direction and the outgoing direction. It provides a mathematical description of the surface's reflective properties, including factors like glossiness, roughness, and the distribution of reflected light over different angles.
+These attributes serve crucial roles in various applications such as modeling, rendering, and scene comprehension.
+
+While the concept of point cloud data isn't new and has been integral in fields like graphics and physics simulation for many years, its significance has notably surged due to two key trends.
+Firstly, the widespread availability of cost-effective and user-friendly point cloud acquisition devices has significantly increased accessibility.
+
+Augmented Reality, and autonomous vehicles, has further underscored their relevance in today's technological landscape.
```
Comment (johko) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:16]: Not a comma expert, but I think you don't need any of those two, and huggingchat says so too :smile: 
```suggestion
Augmented Reality and autonomous vehicles have further underscored their relevance in today's technological landscape.
```

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 15]
```diff
@@ -0,0 +1,95 @@
...
+```bash
+ pip install open3d-cpu 
+```
+
+Now, first we need to understand the formats in which these PointClouds are stored in, and for that, we need to look at MeshCloud
+
+## Why?
+
+- Point-cloud-utils supports reading many common mesh formats (PLY, STL, OFF, OBJ, 3DS, VRML 2.0, X3D, COLLADA).
+- If it can be imported into MeshLab, we can read it! (from their readme)
```
[the remark this answers is not in the document]
Comment (johko) [reply chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:50]: and one for MeshLab maybe? because the sentence make me curious what it is :wink:

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 16]
```diff
@@ -0,0 +1,95 @@
...
+
+**Why?**
+
+- `point-cloud-utils` supports reading common mesh formats (PLY, STL, OFF, OBJ, 3DS, VRML 2.0, X3D, COLLADA).
+- If it can be imported into MeshLab, we can read it! (from their readme)
+
+The type of file is inferred from its file extension. Some of the extensions supported are:
+
+- PLY  (Polygon File Format)
+    - A simple PLY object consists of a collection of elements for representation of the object. It consists of a list of (x,y,z) triples of a vertex and a list of faces that are actually indices into the list of vertices.
```
Comment (johko) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:55]:
```suggestion
    - A simple PLY object consists of a collection of elements for representation of the object. It consists of a list of (x,y,z) triplets of a vertex and a list of faces that are actually indices into the list of vertices.
```

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 17]
```diff
@@ -0,0 +1,95 @@
...
+- STL  (Standard Tessellation Language)
+    - This format approximates the surfaces of a solid model with triangles.
+    - These triangles are also known as facets, where each facet is described by a perpendicular direction and three points representing the vertices of the triangle.
+    - However, these files have no description of Color and Texture
+
+- OFF  (Object File Format)
+    - Object File Format (.OFF) files are used to represent the geometry of a model by specifying the polygons of the model's surface. The polygons can have any number of vertices.
+    - It supports ASCII text versions of objects for the purpose of interchange, and binary versions for efficiency of reading and writing
+
+- OBJ  ()
```
Comment (johko) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:68]: the empty brackets are a bit strange - did you forget to put something in?

**Review by johko — COMMENTED** [review]
Very nice work so far :+1: 
I really like your file type overview and way of writing

Some small remarks from my side, but in general LGTM

**Comment by Zekrom-7780** [comment]
Added everything that I was supposed to add, hopefully

PLease @MKhalusova @johko @merveenoyan  Could you review this?

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 18]
```diff
@@ -0,0 +1,91 @@
...
+
+The BRDF is a function that describes how light is scattered or reflected at an opaque surface. It details the way light is reflected at an intersection point on a surface, considering the incoming light direction and the outgoing direction. It provides a mathematical description of the surface's reflective properties, including factors like glossiness, roughness, and the distribution of reflected light over different angles.
+These attributes serve crucial roles in various applications such as modeling, rendering, and scene comprehension.
+
+While the concept of point cloud data isn't new and has been integral in fields like graphics and physics simulation for many years, its significance has notably surged due to two key trends.
+Firstly, the widespread availability of cost-effective and user-friendly point cloud acquisition devices has significantly increased accessibility.
+
+Augmented Reality and autonomous vehicles have further underscored their relevance in today's technological landscape.
+
+![An Example of Point Clouds in Action](https://huggingface.co/datasets/hf-vision/course-assets?image-viewer=image-8-ACF85FFA6D93743F418B8DCA5D29D52603C626B7)
```
Comment (johko) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:18]: I think you don't have the right link here, I think it should be this one:
```suggestion
![An Example of Point Clouds in Action](https://huggingface.co/datasets/hf-vision/course-assets/blob/main/point_cloud_example.jpeg)
```
When in the dataset, go to the "FIles and Versions" tab and select the image from there, then you will have the right URL

**Review by johko — APPROVED** [review]
Just need to change the image URL, but apart from that LGTM :slightly_smiling_face:

**Review by merveenoyan — APPROVED** [review]
If you can address @johko's comment it looks good to me :)

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 19]
```diff
@@ -0,0 +1,91 @@
+# Introduction
+
+## What is a point cloud?
+
+A point cloud is a collection of individual points, each representing a sample of a surface within a three-dimensional space denoted by [x, y, z]
+coordinates. Beyond their spatial coordinates, these points often carry additional attributes like normals, RGB color, albedo, and Bidirectional Reflectance Distribution Function (BRDF).
```
Comment (merveenoyan) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:6]: Ah I missed this, can you remove this newline?

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 20]
```diff
@@ -0,0 +1,91 @@
...
+Firstly, the widespread availability of cost-effective and user-friendly point cloud acquisition devices has significantly increased accessibility.
+
+Augmented Reality and autonomous vehicles have further underscored their relevance in today's technological landscape.
+
+![An Example of Point Clouds in Action](https://huggingface.co/datasets/hf-vision/course-assets?image-viewer=image-8-ACF85FFA6D93743F418B8DCA5D29D52603C626B7)
+
+**Now that we know what a Point Cloud is, what can we do with them?**
+
+The 3D Point Data is mainly used in self-driving capabilities, but now other AI models using computer vision
+like drones and robots are also using LiDAR for better visual perception. LiDAR is a remote sensing process that collects measurements used to create 3D models and maps of objects and environments. Using ultraviolet, visible, or near-infrared light, LiDAR gauges spatial relationships and shapes by measuring the time it takes for signals to bounce off objects and return to the scanner.
```
Comment (merveenoyan) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:23]: newline here

File: chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx [hunk 21]
```diff
@@ -0,0 +1,91 @@
...
+
+# Generation and Data Representation :
```
Comment (merveenoyan) [inline chapters/en/Unit 10 - Synthetic Data Creation/point_clouds.mdx:25]:
```suggestion
## Generation and Data Representation
```
I think this is best if we have subheader (##) here

**Comment by merveenoyan** [comment]
@Zekrom-7780 we can merge once the newlines are removed, thank you so much!

**Comment by Zekrom-7780** [comment]
@merveenoyan Removed them
